next.config.mjs

```diff
@@ -1,14 +1,6 @@
 const config = {
   reactStrictMode: true,
-  redirects: async () => {
-    return [
-      {
-        source: "/events/xmas-2021-temp",
-        destination: "/events/xmas-2021",
-        permanent: false
-      }
-    ];
-  }
+  unstable_includeFiles: ["assets/data", "public/assets/backgrounds"]
```

Comment:
this doesn't seem to fix the issue yet. Didn't even work in my playground repo where I used `fs.promises.stat()` instead of reading files through `@foxkit/node-util`'s modules.